src/messages/GE.json

```
@@ -359,5 +359,8 @@
   "As user": "(as user)",
   "As Teams app": "(as Teams app)",
   "Use Explorer as sample Teams application": "Use Explorer as sample Teams application",
-  "Use Explorer as logged-in user": "Use Explorer as logged-in user"
+  "Use Explorer as logged-in user": "Use Explorer as logged-in user",
+  "Resource Specific Consent popup": "To use application permissions you must use resource specific consent by downloading the ​",
+  "Application Permissions": "Application Permissions",
+  "Learn more about": "Learn more about"
```

+1 for using the locality messages file :D

---

```suggestion
  "Use Explorer as logged-in user": "Use Explorer as logged-in user",
  "Resource Specific Consent popup": "To use application permissions you must use resource specific consent by downloading the ​",
  "Application Permissions": "Application Permissions",
  "Sample Explorer Teams app": "Sample Explorer Teams app",
  "Learn more about": "Learn more about",
  "resource specific consent": "resource specific consent"
```
Added for some of the above locality changes